packages/bahmni-widgets/src/forms/FormsTable.tsx

```
@@ -73,13 +77,21 @@ const FormsTable: React.FC<WidgetProps> = ({
+        encounterUuids.includes(form.encounterUuid),
+      );
     }
-    return formsData.filter((form) =>
-      encounterUuids.includes(form.encounterUuid),
-    );
-  }, [formsData, encounterUuids]);
+    if (Array.isArray(forms) && forms.length > 0) {
+      result = result.filter((entry) =>
+        forms.some(
+          (f) => f.toLowerCase() === entry.formName.toLowerCase(),
+        ),
```

ESLint

[error] Replace `⏎··········(f)·=>·f.toLowerCase()·===·entry.formName.toLowerCase(),⏎········` with `(f)·=>·f.toLowerCase()·===·entry.formName.toLowerCase()`

(prettier/prettier)